### ISSUE TITLE
Error L107 findline

### ISSUE THREAD

**Issue by cerinawithasea** [romanoh/Mp3tag-Repository#23]
I had to reformat both my laptops and I have mp3tag on both machines (win and mac) 
If i had to fix this before i do not remember but the scrapers that are HTML updated (wild speculation) give me this 
![Untitled](https://github.com/romanoh/Mp3tag/assets/71559155/c5ac40fd-6280-4fc1-b3e2-d95a1ce69948)
I remember before you were so helpful and i thought it just best to try and fix it instead of ignoring it and using something else.  the search is so fast. Weirdly the part of the book community I am in doesn't really seem to do imbedded tags (insane i know) so im over here bugging you again
If you have the extra time i was hoping you could help me fix this 
I have the newest releases of mp3tag on both machines (i wanted to ask is that the best think to use in your opinion) I like it but i guess you never know what you are missing
As always I appreciate you
Cerina
quick edit
The screenshot is from the storytell src buts it is all the same

**Comment by romanoh** [comment]
witch script it is? and in mac i dont know mutch

**Comment by cerinawithasea** [comment]
the scripts seem to work cross OS's I started using your stuff on windows years ago when i was following a guide for plex and just brought it over to mac when i got one.  unfortunately in Mac you do not have as much control ,  can really only import src files
I just got auto_m4b set up with the docker (is that something you use they dont use mp3tag for the tagging so i dont really take full use of it)
now that my mind isnt so flustered its the json (some are labeled that) not html or whatever i said


For that screenshot it was storytell
i jut tried the honeyeaters on storytell and got 107 Again
ya know what the more i mess with these ...they are working  sometimes (but not json..maybe. ill keep better track on both machines)
I think when i saw these before i got the error and just didn't investigate and moved on to another
would it help if i kept track of line errors and the selection i picked?

I also got same error but L458 in the attached trying to pull open 
![image](https://github.com/romanoh/Mp3tag/assets/71559155/0371d289-60d3-47d1-9041-d4ecc89f3668)

 (note i did tiny change that for a full date from audible way down here
findline ", \"datePublished\": \""
		unspace
		findinline ": \""
		outputto "Year"
		SayNChars 10
